cluster.go

```diff
@@ -937,7 +937,11 @@ func (c *Cluster) allNodesReady() bool {
 func (c *Cluster) handleNodeAction(nodeAction nodeAction) error {
 	j, err := c.generateResizeJob(nodeAction)
 	if err != nil {
-		return err
+		c.logger().Printf("generateResizeJob error: err=%s", err)
+		if err := c.setStateAndBroadcast(ClusterStateNormal); err != nil {
+			c.logger().Printf("setStateAndBroadcast error: err=%s", err)
+		}
+		return c.setStateAndBroadcast(ClusterStateNormal)
```

Comment:
what's the reason for calling `c.setStateAndBroadcast(ClusterStateNormal)` again?

Reply:
Mistake. Meant to just return the error we get above. Patching now.